packages/reactotron-core-client/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "noImplicitAny": false, // Might be worth setting to true after we get good typing in
+        "removeComments": false,
+        "preserveConstEnums": true,
+        "sourceMap": true,
+        "allowSyntheticDefaultImports": true,
```

Comment:
Currently experiencing this problem on my client project with a few libs.  I can `import * as ValidateJS from 'validate.js'`, but it won't work in the `jest` environment.  I think somethings borked with the typings on that library.

But I digress.